config/locales/en.yml

```diff
@@ -25,7 +28,9 @@ en-GB:
       processed_on: Date processed
       form_name: Form
       fee: Fee
+      part_payment: Part- payment
```

Comment:
Good catch... originally yes, to force a wrap of the text, before Clive worked his magic...